BaikalIO/material_io.cpp

```diff
@@ -468,14 +486,14 @@ namespace Baikal
                 }
                 else
                 {
-                    std::ostringstream oss;
-                    oss << (std::uint64_t)texture.get() << ".jpg";
+                    //std::ostringstream oss;
+                    //oss << (std::uint64_t)texture.get() << ".jpg";
```

Comment:
1) Remove dead code
2) It's ok for me to switch to names instead of adresses, but please make all names lower-case.